### PR TITLE
Add "Try Lighthouse" page

### PR DESCRIPTION
**Pull request by ebidel**
POC for now to see how things can work on staging. Purposefully not creating any pointer links. 

This also updates our template files to set a correct meta viewport.

### CODE REVIEW

File: gae/404.tpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <title>Page not found</title>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
-    <meta name="viewport" content="initial-scale=1, minimum-scale=1, width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```
Comment (petele) [inline gae/404.tpl]: To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`

File: gae/500.tpl [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <title>Server Error</title>
     <meta charset="utf-8">
     <meta name="robots" content="noindex, nofollow">
-    <meta name="viewport" content="initial-scale=1, minimum-scale=1, width=device-width">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```
Comment (petele) [inline gae/500.tpl]: To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`

File: gae/article.tpl [hunk 3]
```diff
@@ -2,9 +2,10 @@
 <html lang="{{ lang }}">
   <head>
     <meta charset="utf-8" />
+    <meta name="robots" content="noindex">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```
Comment (petele) [inline gae/article.tpl]: To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`

File: gae/home.tpl [hunk 4]
```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8" />
     <meta name="robots" content="noindex">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
```
Comment (petele) [inline gae/home.tpl]: To be consistent with DevSite, let's use `<meta name="viewport" content="width=device-width, initial-scale=1">`

File: src/content/en/tools/lighthouse/run.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Run Lighthouse against any site.
+
+{# wf_updated_on: 2017-06-06 #}
+{# wf_published_on: 2016-06-06 #}
```
Comment (petele) [inline src/content/en/tools/lighthouse/run.md]: update `wf_published_on` to this year.

File: src/content/en/tools/lighthouse/run.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+}
+.framebox {
+  margin: 0;
+}
+</style>
+
+Take [Lighthouse](/web/tools/lighthouse/) for a spin. Enter the URL of any site
+to get started.
+
+{% framebox height="500px" %}
```
Comment (petele) [inline src/content/en/tools/lighthouse/run.md]: Any reason you're putting this in a framebox and not just a simple iframe?

Comment (ebidel) [inline src/content/en/tools/lighthouse/run.md]: Nope! Will update.

**Comment by ebidel** [comment]
PTAL

**Pull request merged**